bundles/org.openhab.binding.kostalinverter/src/main/java/org/openhab/binding/kostalinverter/internal/secondgeneration/SecondGenerationHandler.java

```diff
@@ -151,6 +151,7 @@ public void handleCommand(ChannelUID channelUID, Command command) {
 
     @Override
     public void initialize() {
+        Thread.currentThread().interrupt();
```

Comment:
Why are you adding this instruction at this place ?

Reply:
I thought you ment this place as " first instruction in the block". Please, advice me were you want me to put it.

Reply:
I found some information about Thread.... and have tried to use it in a way I also found in another binding, could maybe  be a solution.